src/Modules/ElementUtils/inheritance.js

```diff
@@ -6,13 +6,13 @@ const INHERITED_MEMBER_QUERY = {
   $or: [
     {
       'meta.links': {
-        href: 'http://example.com/inherited-member/',
+        href: 'http://refract.link/inherited-member/',
```

Comment:
It's in the commit message - https://github.com/apiaryio/attributes-kit/pull/449/commits/6f8d36882304cbe5987df9e51c59c148a9dd8aae

Eidolon relies on this string